applications/ila-cookie-banner/RELEASE.md

```diff
@@ -0,0 +1,94 @@
+# Cookie Banner Release Process
+
+This is a guide for installing the a new release of the campus cookie banner to the content delivery network (CDN) server S3 bucket.
```

Comment:
Please at least remove the redundant `the`. I also think the wording at the end of the sentence is akward:

```suggestion
This is a guide for installing a new release of the campus cookie banner to the Content Delivery Network (CDN).
```